src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/DeploymentParameters.cs

```diff
@@ -99,6 +99,8 @@ public DeploymentParameters(DeploymentParameters parameters)
             }
         }
 
+        public TestApplication TestApplication { get; set; }
```

Comment:
This object model is circular. DeploymentParamenters has a TestApplication, but TestApplication methods take DeploymentParmameters.